drivers/spi/spi_mcux_lpspi.c

```diff
@@ -75,6 +75,7 @@ struct spi_mcux_data {
 
 #ifdef CONFIG_SPI_RTIO
 	struct rtio *r;
+	struct mpsc io_q;
```

Comment:
The RTIO `#ifdef` gives a hint that this is part of RTIO. Good now that `struct rtio_mpsc` became `struct mpsc`.

Reply:
Is this comment suggesting that 
struct mpsc io_q;
should not be in the CONFIG_SPI_RTIO check?

Reply:
Sorry if that was not clear, I simply pointed that it was a good thing that it was still in the `CONFIG_SPI_RTIO` check.